connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -427,4 +435,62 @@ public void ensureValid(String name, Object value) {
         }
     }
 
+    private static class ResponseHttpHeadersValidator implements ConfigDef.Validator {
+        @Override
+        public void ensureValid(String name, Object value) {
+            String strValue = (String) value;
+            if (strValue.isEmpty()) {
+                return;
+            }
+
+            String[] configs = StringUtil.csvSplit(strValue);
```

Comment:
~How about using `strValue.split(',')`, and avoiding the need to import `org.eclipse.jetty.util.StringUtil`?~

Okay, I see now that `cvsSplit` handles quotes. How about a comment:
```suggestion
            String[] configs = StringUtil.csvSplit(strValue); // handles and removes surrounding quotes
```